x/oracle/types/msgs.go

```diff
@@ -74,6 +82,12 @@ func (msg MsgRequestData) ValidateBasic() error {
 	if len(msg.ClientID) > MaxClientIDLength {
 		return WrapMaxError(ErrTooLongClientID, len(msg.ClientID), MaxClientIDLength)
 	}
+	if msg.PrepareGas <= 0 {
+		return sdkerrors.Wrapf(ErrInvalidVersion, "invalid prepare gas: %d", msg.PrepareGas)
```

Comment:
ErrInvalidOwasmGas

Reply:
please add test for this error

Reply:
yep, I'm already append test for this in `TestMsgRequestDataValidation`